lob/api_requestor.py

```diff
@@ -72,7 +73,9 @@ def request(self, method, url, params=None):
                 del params['query']
 
             for k, v in params.items():
-                if isinstance(v, dict) and not isinstance(v, lob.resource.LobObject):
+                if k == 'merge_variables':
+                    explodedParams[k] = json.dumps(v)
```

Comment:
This is not a long term solution but after spending a good amount of time trying to figure out how to nest dictionaries and lists for multipart requests this is the only working thing I could think of. A possible long term solution might be this [lib](https://docs.aiohttp.org/en/stable/multipart.html).

Reply:
This should be ok for now. I was playing around with the requests library we use and it had some weird encoding behavior. Since we allow this within our API this should be good for now.